backend/danswer/connectors/gong/connector.py

```diff
@@ -280,9 +289,12 @@ def poll_source(
         # so adding a 1 day buffer and fetching by default till current time
         start_one_day_offset = start_datetime - timedelta(days=1)
         start_time = start_one_day_offset.isoformat()
+
+        # set end_time to `None` if in the `end` is <= 24 hours ago (e.g.
```

Comment:
Why is this a thing specific for Gong?
we could just do it across the board (but maybe 24 hours is a bit long?)